core/src/main/java/org/aerogear/mobile/core/bootstrap/AeroGearBootstrap.java

```diff
@@ -8,12 +8,24 @@
 import android.support.annotation.Nullable;
 
 import org.aerogear.mobile.core.MobileCore;
+import org.aerogear.mobile.core.exception.ConfigurationNotFoundException;
+import org.aerogear.mobile.core.metrics.MetricsService;
 
 public class AeroGearBootstrap extends ContentProvider {
 
     @Override
     public boolean onCreate() {
         MobileCore.init(getContext());
+
+        try {
+            MetricsService metricsService =
+                            MobileCore.getInstance().getService(MetricsService.class);
+            metricsService.sendAppAndDeviceMetrics(
+                            error -> MobileCore.getLogger().error(error.getMessage()));
+        } catch (ConfigurationNotFoundException e) {
+            // Metrics is not enable and should be ignored
```

Comment:
`Metrics SDK is not enable*d*. Ignoring error`

Reply:
Replaced by your suggestion as a Logger